### PR TITLE
python: remove "dbus_internal" hack

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
Remove the special-casing of the "internal" dbus channel and implement it as a peer-to-peer connection inside of the bridge.

The other side of this connection lives inside of a new file, `internal_endpoints.py`, which contains the objects that we want to appear on the internal D-Bus.

Most of these are still stubbed out, but the parts that were implemented before have been copied over.

Bump our dependency on systemd_ctypes for the required APIs for peer-to-peer connections and object exporting.

### CODE REVIEW

File: src/cockpit/internal_endpoints.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+@bus.Object.interface('cockpit.Packages')
+class PackagesEndpoint(bus.Object):
+    ...
+
+
+@bus.Object.interface('cockpit.Superuser')
+class SuperuserEndpoint(bus.Object):
+    @bus.Object.method(in_types=['s'])
+    def start(self, _bridge):
+        ...
```
Comment (martinpitt) [inline src/cockpit/internal_endpoints.py:62]: Hmm, why doesn't this make the pycheck scenario burst up in flames? This file is full of `...`

Comment (martinpitt) [reply src/cockpit/internal_endpoints.py:62]: `<class 'ellipsis'>` what on earth is that now? Sometimes it feels like Python is getting worse..

**Review by martinpitt — APPROVED** [review]
Thanks!

File: src/cockpit/internal_endpoints.py (hunk 1 above)
Comment (allisonkarlitskaya) [reply src/cockpit/internal_endpoints.py:62]: `...` is a conventional alternative for "I'll implement this later"

It's basically equivalent to a docstring, or `pass`.

**Pull request merged**